WordPressLoginFlow/build.gradle

```diff
@@ -31,18 +35,21 @@ android {
 }
 
 dependencies {
-    implementation ('org.wordpress:utils:1.20.3') {
+    implementation ('org.wordpress:utils:1.22') {
         exclude group: "com.android.volley"
     }
 
-    implementation 'androidx.appcompat:appcompat:1.0.2'
-    implementation 'androidx.vectordrawable:vectordrawable-animated:1.0.0'
-    implementation 'androidx.media:media:1.0.1'
+    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'androidx.vectordrawable:vectordrawable-animated:1.1.0'
+    implementation 'androidx.media:media:1.1.0'
     implementation 'androidx.legacy:legacy-support-v13:1.0.0'
     implementation 'androidx.gridlayout:gridlayout:1.0.0'
     implementation 'com.google.android.material:material:1.1.0'
 
-    api 'com.google.android.gms:play-services-auth:15.0.1'
+    implementation "androidx.core:core-ktx:1.2.0"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+
+    api 'com.google.android.gms:play-services-auth:18.0.0'
```

Comment:
I'm kinda worried about this version bump, as it seems to be quite a big jump from `15.0.1` to `18.0.0`. Are we sure there aren't any breaking changes between these?

Everything else seems to be fine.